Makefile

```diff
@@ -124,19 +124,22 @@ test-cpp:
 setup-cpp-full-mod: 
 	sudo apt install -y valgrind gdb
 
+setup-cpp-debug: 
+	sudo apt install -y valgrind gdb
+
 # Linux only
 test-cpp-full-mod-asan: build-asan
-	viam-cartographer/build/unit_tests -p -l all -t CartoFacade_io -t CartoFacadeCPPAPI
+	viam-cartographer/build/unit_tests -p -l all
 
 test-cpp-full-mod-valgrind: build-debug
```

Comment:
In a follow up PR I intend to go back & change the workflows to run C++ tests in valgrind without the make target having full-mod in the name